app/src/main/java/org/oppia/app/player/state/StateFragmentPresenter.kt

```diff
@@ -354,6 +357,46 @@ class StateFragmentPresenter @Inject constructor(
     itemList.add(contentViewModel)
     stateAdapter.notifyDataSetChanged()
   }
+  private fun checkAndAddInteraction() {
```

Comment:
Nit: Please add a blank newline before this method for consistency.

Reply:
Done.